### DISCUSSION TITLE
Lazer Gerbl no connect

### DISCUSSION THREAD

**Discussion by ghost** [arkypita/LaserGRBL#2019] [Q&A]
CH 340 shows in my com/lpt  error connecting your engraver and I have not the slightest as to how to fix this, I have tried everything that I know to no avail

**Comment by Tribo-2dot0** [comment]
Baud 115200?

**Reply by ghost** [reply]
Tried both 57 and 115, I can get to pictures one with just lines and one with a picture t1 using com3 but it will not connect. Is my USB the com port? also the driver installs just fine but where should the driver go?

**Reply by Tribo-2dot0** [reply]
Verifique no Gerenciador de Dispositivos do Windows.
